manipulation/build.gradle.kts

```diff
@@ -43,6 +43,7 @@ dependencies {
 
     testRuntimeOnly("commons-io:commons-io:${project.extra.get("commonsVersion")}")
     testImplementation("junit:junit:${project.extra.get("junitVersion")}")
+    testImplementation("org.apache.commons:commons-lang3:${project.extra.get("commonsLang3Version")}")
```

Comment:
This version (3.5) is already on the classpath due to some other dependency, but not explicitly required. It is needed because 2.x which we do already include does not contain the `equalsAny` method.

Reply:
If this is *already* on the (test runtime) classpath why do we need to explicitly state it?